.github/pull_request_template.md

```diff
@@ -1,24 +1,30 @@
-## :pushpin: Summary
+### :pushpin: Summary
 
 <!-- If merged, this PR.... 
 This should be a short TL;DR that includes the purpose of the PR.
 -->
 
-## :hammer_and_wrench: Detailed description
+### :hammer_and_wrench: Detailed description
 
 <!-- If more details are appropriate, add them here. What code changed, and why? -->
 
-## :camera_flash: Screenshots
+### :camera_flash: Screenshots
 
 <!-- Screenshots always help, especially if this PR will change what renders to the browser -->
 
-## :link: External links
+### :link: External links
 
 <!-- Issues, RFC, etc. -->
 
 ***
 
-## :white_check_mark: Reviewer's checklist
+### 👀 How to review
```

Comment:
[thought] for me this is usually obvious based either on the person who wrote the PR or a quick look at the commits tab. having to indicate this in every PR seems like it might get tedious, and also "how to review" is more up to the reviewer in my opinion than the person who wrote the PR.

Reply:
No objections 🚢